src/CalcViewModel/GraphingCalculator/GraphingSettingsViewModel.h

```diff
@@ -226,6 +227,7 @@ namespace CalculatorApp::ViewModel
                 if (value && m_Graph != nullptr && m_Graph->TrigUnitMode != (int)Graphing::EvalTrigUnitMode::Radians)
                 {
                     m_Graph->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Radians;
+                    TraceLogger::GetInstance()->LogGraphSettingsChanged(GraphSettingsType::TrigUnits);
```

Comment:
Very nit: let's call this method after the 3 RaisePropertyChanged to update the UI as fast as possible.